src/pulsar_spectra/catalogue_papers/Bell_2016_raw_to_yaml.py

```diff
@@ -1,10 +1,17 @@
 import yaml
 
+from pulsar_spectra.scripts.csv_to_yaml import dump_yaml
+
 with open("Bell_2016_raw.txt", "r") as raw_file:
     lines = raw_file.readlines()
     print(lines)
 
-pulsar_dict = {}
+pulsar_dict = {
+    "Paper Metadata": {
+        "Data Type": "Beamforming",
+        "Observation Span": "Single-epoch",
```

Comment:
This paper I think falls into the "Multi-epoch" category; it is long-term monitoring (imaging) with the MWA.